config/content_items.yml

```diff
@@ -34,10 +34,6 @@ special_routes:
     base_path: "/account/home"
     title: "Account home page"
     rendering_app: "frontend"
-  - content_id: "19c4cef8-d0ad-483b-93ea-e07f7ed4734a"
-    base_path: "/account/cookies-and-feedback"
```

Comment:
Does this need to go at the same time?
https://github.com/alphagov/frontend/blob/6ad2e28c2dfa11357d2fe2b6b37a06406e011b24/test/functional/account_cookies_and_feedback_controller_test.rb

Reply:
Also 🤔 about the special route publication here... we may need to recall these and publish a redirect?

Reply:
Those are removed in https://github.com/alphagov/frontend/pull/3225 

Not sure about the redirect. In theory we've removed all places that link to this page, but we could publish a redirect just to be safe.